build/source/driver/multi_driver.f90

```diff
@@ -57,44 +57,63 @@ program multi_driver
 USE summaFileManager,only:LOCALPARAM_INFO,BASINPARAM_INFO   ! files defining the default values and constraints for model parameters
-USE data_struc,only:doJacobian                              ! flag to compute the Jacobian
-USE data_struc,only:localParFallback                        ! local column default parameters
-USE data_struc,only:basinParFallback                        ! basin-average default parameters
-USE data_struc,only:mpar_meta,bpar_meta                     ! metadata for local column and basin-average model parameters
-USE data_struc,only:numtim                                  ! number of time steps
-USE data_struc,only:time_data,time_hru,refTime              ! time and reference time
-USE data_struc,only:forc_data,forc_hru                      ! model forcing data
-USE data_struc,only:type_data,type_hru                      ! classification of veg, soils etc.
-USE data_struc,only:attr_data,attr_hru                      ! local attributes (lat, lon, elev, etc.)
-USE data_struc,only:mpar_data,mpar_hru                      ! local column model parameters
-USE data_struc,only:mvar_data,mvar_hru                      ! local column model variables
-USE data_struc,only:indx_data,indx_hru                      ! local column model indices
-USE data_struc,only:bpar_data                               ! basin-average model parameters
-USE data_struc,only:bvar_data                               ! basin-average model variables
-USE data_struc,only:model_decisions                         ! model decisions
-USE data_struc,only:urbanVegCategory                        ! vegetation category for urban areas
-USE data_struc,only:globalPrintFlag                         ! global print flag
+! provide access to global data
+USE globalData,only:refTime                                 ! reference time
+USE globalData,only:startTime                               ! start time
+USE globalData,only:finshTime                               ! end time
+USE globalData,only:doJacobian                              ! flag to compute the Jacobian
+USE globalData,only:localParFallback                        ! local column default parameters
+USE globalData,only:basinParFallback                        ! basin-average default parameters
+USE globalData,only:time_meta                               ! metadata for time information 
+USE globalData,only:mpar_meta,bpar_meta                     ! metadata for local column and basin-average model parameters
+USE globalData,only:indx_meta,prog_meta,diag_meta,flux_meta ! metadata for local column variables
+USE globalData,only:averageFlux_meta                        ! metadata for time-step average fluxes
+USE globalData,only:numtim                                  ! number of time steps
+USE globalData,only:model_decisions                         ! model decisions
+USE globalData,only:urbanVegCategory                        ! vegetation category for urban areas
```

Comment:
Why is `urbanVegCategory` called out as a separate global? Does that belong in this PR, which is about refactoring the numerix?